README.md

```diff
@@ -1,4 +1,4 @@
-First things first. You're going to need to get a developer key from Careerbuilder. You can obtain a key at http://api.careerbuilder.com/RequestDevKey.aspx.
+First things first. You're going to need to get a developer key from Careerbuilder. Developer keys are available to Partners. You can fill out the form at http://developer.careerbuilder.com/partner_messages/new to become a partner.
```

Comment:
Wording looks good to me. Tiny nitpick - 'partner' has inconsistent capitalization.